fast/stages/0-bootstrap/data/custom-roles/ngfw_enterprise_admin.yaml

```diff
@@ -40,3 +40,8 @@ includedPermissions:
   - networksecurity.securityProfiles.list
   - networksecurity.securityProfiles.update
   - networksecurity.securityProfiles.use
+  - networksecurity.tlsInspectionPolicies.create
```

Comment:
Nit: reference in README that module.branch-nsec-sa[0].iam_email and local.principals.gcp-network-admins can create/update TLS inspection policies in the security project, which while reasonable might be unexpected to some.